workflow/rules/common.smk

```diff
@@ -64,7 +68,7 @@ def generate_read_group(wildcards):
         "{}.{}".format(wildcards.sample, wildcards.lane),
         "{}_{}".format(wildcards.sample, wildcards.type),
         get_unit_platform(units, wildcards),
-        "{}.{}.{}".format(get_unit_flowcell(units, wildcards), wildcards.lane, get_unit_barcode(units, wildcards)),
+        "{}.{}.{}".format(get_unit_flowcell(units, wildcards), wildcards.lane, get_unit_barcodes(units, wildcards)),
     )
```

Comment:
In the parabricks module the LB flag is also set:
"{}_{}".format(unit.sample, unit.type),

Reply:
added